workflow/controller/workflowpod.go

```diff
@@ -259,6 +259,14 @@ func (woc *wfOperationCtx) createWorkflowPod(ctx context.Context, nodeName strin
 	// container's PID and root filesystem.
 	pod.Spec.Containers = append(pod.Spec.Containers, mainCtrs...)
 
+	// Add this annotation in order to specify the default container that will be targeted with commands like "kubectl exec/logs"
+	if tmpl.GetType() != wfv1.TemplateTypeContainerSet {
+		pod.ObjectMeta.Annotations[common.AnnotationKeyDefaultContainer] = common.MainContainerName
```

Comment:
minor - you do not need `.ObjectMeta`

Reply:
What did you mean by that? Not sure I understood. Annotations map is sitting inside of the ObjectMeta, isn't it?
I don't have much experience with golang, so let me know please if it's something with language syntax.

Reply:
you can use `pod.Annotations`

Reply:
Got you, thanks. Not a big deal, but across the workflowpod.go file `Annotations` are accessed via `pod.ObjectMeta` object. Maybe it's better to keep it the same? If so, should I change my code or the one that uses `ObjectMeta.Annotations`?

Reply:
@alexec WDYT regarding my comment?